corehq/apps/hqwebapp/static/hqwebapp/js/hq.helpers.js

```diff
@@ -148,10 +148,15 @@ hqDefine("hqwebapp/js/hq.helpers", [
         beforeSend: function (xhr, settings) {
             // Don't pass csrftoken cross domain
             // Ignore HTTP methods that do not require CSRF protection
-            if (!/^(GET|HEAD|OPTIONS|TRACE)$/.test(settings.type) && !this.crossDomain) {
-                var $csrf_token = $("#csrfTokenContainer").val();
-                xhr.setRequestHeader("X-CSRFToken", $csrf_token);
+            if (!/^(GET|HEAD|OPTIONS|TRACE)$/.test(settings.type)) {
+                if (!this.crossDomain) {
```

Comment:
Won't this also be true for Formplayer requests since they are mapped to a path on the same domain. I would think we'd want the same protection for the Formpalyer CSRF token.